beginner_source/nlp/word_embeddings_tutorial.py

```diff
@@ -1,158 +1,135 @@
 # -*- coding: utf-8 -*-
 r"""
-Word Embeddings: Encoding Lexical Semantics
+단어 임베딩: 어휘의 의미를 인코딩하기
 ===========================================
-
-Word embeddings are dense vectors of real numbers, one per word in your
-vocabulary. In NLP, it is almost always the case that your features are
-words! But how should you represent a word in a computer? You could
-store its ascii character representation, but that only tells you what
-the word *is*, it doesn't say much about what it *means* (you might be
-able to derive its part of speech from its affixes, or properties from
-its capitalization, but not much). Even more, in what sense could you
-combine these representations? We often want dense outputs from our
-neural networks, where the inputs are :math:`|V|` dimensional, where
-:math:`V` is our vocabulary, but often the outputs are only a few
-dimensional (if we are only predicting a handful of labels, for
-instance). How do we get from a massive dimensional space to a smaller
-dimensional space?
-
-How about instead of ascii representations, we use a one-hot encoding?
-That is, we represent the word :math:`w` by
+**번역**: `임성연 <http://github.com/sylim2357>`_
+
+단어 임베딩(word embedding)이란 코퍼스(corpus) 내 각 단어에 일대일로 대응하는 밀집된 실수 벡터(dense vector)의 집합, 혹은 이 벡터를
+구하는 행위를 가리킵니다. 주로 단어를 피처(feature)로 사용하는 자연어 처리 분야에서는 단어를 컴퓨터 친화적인
+형태로 바꾸어 주는 작업이 필수적입니다. 컴퓨터가 단어를 바로 이해하기는 상당히 어렵기 때문이죠.
+그렇다면, 단어를 어떻게 표현하는 것이 좋을까요? 물론 각 문자에 해당하는 ASCII코드를 사용할 수 있겠지만,
+ASCII코드는 이 단어가 *무엇*인지를 알려줄 뿐, 단어가 어떤 *의미*를 가지는지는 알려주지 않습니다.
```

Comment:
Sphinx에서는 앞뒤로 공백이 있어야 문법을 인식하고 있습니다. (아래는 `make html-noplot` 으로 빌드한 결과물로, 공백이 필요합니다..)

![image](https://user-images.githubusercontent.com/9343724/77766851-8b668580-7083-11ea-8dfe-e12e44e5cc73.png)

아래와 같은 식은 어떨까요?
```suggestion
ASCII코드는 이 단어가 *무엇* 인지를 알려줄 뿐, 단어가 어떤 *의미* 를 가지는지는 알려주지 않습니다.
```